src/components/order-service/order-service.controller.ts

```diff
@@ -164,6 +249,75 @@ export class OrderServiceController implements angular.IController {
     }
   }
 
+  private updateBindability() {
+    this.bindStep.hidden = _.size(this.ctrl.applications) < 1;
```

Comment:
Should probably fix this in a follow up PR but there is an existing issue https://github.com/openshift/origin-web-console/issues/1470 around binding even if you have no applications, the bind dialog is incorrect today (it actually shows up always in the overview but gives you the option to pick apps when there are none).  We should always let you create the generic secret binding, even if you don't have applications.